cmd/frontend/enterprise/enterprise.go

```diff
@@ -31,6 +31,7 @@ type Services struct {
 	LicenseResolver           graphqlbackend.LicenseResolver
 	DotcomResolver            graphqlbackend.DotcomRootResolver
 	SearchContextsResolver    graphqlbackend.SearchContextsResolver
+	EnterpriseResolver        graphqlbackend.EnterpriseResolver
```

Comment:
I think the previous naming was more consistent with the conventions applied in `enterprise` folders.

E.g. the folder is `/enterprise/`, which should mean, that everything in the folder is related to enterprise stuff. Calling the resolver `EnterpriseResolver` does not tell me anything about what type of data it is returning. Whereas if I look at other resolver type names, it's immediately obvious.

In the future, when we add more functionality to this resolver, we can rename it accordingly. Now it seems to me like a premature generalisation.

```suggestion
	OrgRepositoryResolver        graphqlbackend. OrgRepositoryResolver
```

Reply:
Makes sense! I'll revert the rename.